server/routes/handlers/getAssignUserToOutcomeRequestHandler.js

```diff
@@ -0,0 +1,9 @@
+const getAssignUserToOutcomeRequestHandler = caseService => async (req, res) => {
+  const { params: { hearingId }, session, body: { targetDefendantId: defendantId, targetCourtCode: courtCode } } = req
+
+  await caseService.assignHearingOutcome(hearingId, res.locals.user.name)
+  session.assignHearingOutcomeSuccess = true
```

Comment:
This needs to be unset in the case summary handler, otherwise this will not be cleared and keeps rendering the notification on case summary page